it/uk/gov/hmrc/agentsubscriptionfrontend/connectors/AgentAssuranceConnectorISpec.scala

```diff
@@ -62,4 +64,23 @@ class AgentAssuranceConnectorISpec extends UnitSpec with OneAppPerSuite with Wir
       }
     }
   }
+
+  "hasActiveCesaRelationship" should {
```

Comment:
pls, add here monitoring assertions, i.e. [like this ones](https://github.com/hmrc/agent-access-control/blob/f69ef67e8c2a15587e04c86f1325446421e9f3b7/it/uk/gov/hmrc/agentaccesscontrol/connectors/desapi/DesAgentClientApiConnectorISpec.scala#L164)